docs/src/development/transfer-dedicated.md

```diff
@@ -1,149 +1,135 @@
 ---
-title: "Transferring data to and from a Dedicated cluster"
+title: "Transfer data to and from a Dedicated cluster"
 weight: 13
-sidebarTitle: "Syncing to Dedicated"
+sidebarTitle: "Sync to Dedicated"
 ---
 
-## Backing up staging and production files
+Transferring data to and from a Dedicated cluster slightly differs from the grid.
+For advanced command line utilities, you might need granular SSH access to a specific host.
```

Comment:
:hammer_and_wrench: I would probably remove this line. It seems a little vague (it's unclear whether I'll need it or not and what exactly I'll need) and I'm not sure what it adds at the start.